repos/fdbt-site/src/pages/searchOperators.tsx

```diff
@@ -23,70 +25,78 @@ export interface SearchOperatorProps {
     searchText: string;
     errors: ErrorInfo[];
     searchResults: Operator[];
-    selectedOperators: Operator[];
+    preSelectedOperators: Operator[];
     csrfToken: string;
 }
 
-export const showSelectedOperators = (selectedOperators: Operator[], errors: ErrorInfo[]): ReactElement => {
+export const showSelectedOperators = (
+    selectedOperators: Operator[],
+    setSelectedOperators: React.Dispatch<React.SetStateAction<Operator[]>>,
+    errors: ErrorInfo[],
+): ReactElement => {
     const removeOperatorsErrors: ErrorInfo[] = [];
     errors.forEach((err) => {
         if (err.id === removeOperatorsErrorId) {
             removeOperatorsErrors.push(err);
         }
     });
+    const removeOperator = (nocCode: string) => {
+        if (nocCode === 'all') {
```

Comment:
perhaps using a flag / extra parameter called removeAll and passing true then doing a check if(removeAll){

is better because what if you have an operator call 'all'